sdk/metricsadvisor/ai-metrics-advisor/test/public/datafeed.spec.ts

```diff
@@ -313,8 +313,9 @@ matrix([[true, false]] as const, async (useAad) => {
             viewerEmails: ["viewer1@example.com"],
             actionLinkTemplate: "Updated Azure Blob action link template"
           };
-          const updated = await client.updateDataFeed(createdAzureBlobDataFeedId, patch);
-
+          await client.updateDataFeed(createdAzureBlobDataFeedId, patch);
+          //assert.equal(updatedResponse._response.status, 200||204);
```

Comment:
@jeremymeng Can we do a similar check on these lines - checking if either of two values are valid?

Reply:
Not necessary. Any other status code would be a thrown error.

Reply:
You can try it out, maybe trying to update some field that cannot be updated?

Reply:
or passing in some invalid values

Reply:
```suggestion
   ```